app/react/src/server/config/webpack.config.prod.js

```diff
@@ -29,7 +29,7 @@ export default function() {
           screw_ie8: true,
           warnings: false,
         },
-        mangle: false,
+        mangle: true,
```

Comment:
Do you have an actual non-theoretical example @joscha ?

This saves A LOT of bytes in the production bundle, which is one of the issues I need to resolve to get automatic example deployment working.

I'm gonna go ahead with this unless someone can provide me a non-theoretical reason not to do this.